src/collections/service-mesh-workshops/2020_05_12_open_source_101/index.mdx

```diff
@@ -2,7 +2,7 @@
 title: "Open Source 101 at Home"
 abstract: "Solving the Service Mesh Adopter’s Dilemma - Open Source 101 at Home 2020"
 thumbnail: "./open_source_101_original.jpg"
-eurl: https://opensource101.com/sessions/solving-the-service-mesh-adopters-dilemma/
+
```

Comment:
Should we place the `https://opensource101.com/` URL with the YT substitute? It is 'https://www.youtube.com/watch?v=Q1zSWbO0RmI'

Reply:
Don't add a blank line instead.